pkg/scheduler/scheduler.go

```diff
@@ -358,6 +358,11 @@ func (s *Scheduler) doScheduleBinding(namespace, name string) (err error) {
 		return nil
 	}
 
+	if util.IsBindingSuspendScheduling(rb) {
+		klog.V(4).InfoS("Skip schedule suspended ResourceBinding", "ResourceBinding", klog.KObj(rb))
+		return nil
+	}
```

Comment:
I think we should skip reconciling the suspended ResourceBinding earlier. 
Probably at https://github.com/karmada-io/karmada/blob/59b8835bda0710c42e2ae30f6c13c13f2a513480/pkg/scheduler/event_handler.go#L93-L113

Reply:
seems OK.